src/bootstrap/tool.rs

```diff
@@ -81,10 +81,7 @@ impl Step for ToolBuild {
             }
             let (id, features, filenames) = match msg {
                 compile::CargoMessage::CompilerArtifact {
-                    package_id,
-                    features,
-                    filenames,
-                    target: _,
+                    package_id, features, filenames, ..
```

Comment:
Please don't make this change, since then changes to the enum don't cause compiler errors.